### PR TITLE
Spin64 patch 1

### PR DESCRIPTION
**Pull request by spin64**

### CODE REVIEW

**Comment by jaihysc** [comment]
Here are my suggestions looking at the website:
- Select a more appropriate background for the home page, blank is fine
- Change the font to a sans-serif font
- The logo should be clickable to take one back to the home page; with that the "Home" button is unnecessary
- Move the "Change theme" button into the navbar instead of sitting at an odd location on the screen

**Comment by jaihysc** [comment]
The media folder is with a lowercase m, this is important!

**Review by jaihysc — CHANGES_REQUESTED** [review]
Website visuals:
- Select a more appropriate background for the home page, blank is fine
- Change the font to a sans-serif font
- The logo should be clickable to take one back to the home page; with that the "Home" button is unnecessary
- Move the "Change theme" button into the navbar instead of sitting at an odd location on the screen

File: Help.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                <li><a href="https://forms.gle/QmQRTCUmqyDopNDF7" class="menu">Report</a></li>
+                <button type="button" id="theme_changer" class="button">Change theme</button>
+            </ul>
+        </nav>
+
+        <p>This is the help page</p>
+        <p>You can navigate to other parts of the website by pressing the menu buttons at the top on the navigation bar</p>
+        <p>The home page is just for show</p>
+        <p>The browse page is where you can browse the archived videos/pictures</p>
+        <p>THe feed back page sends you to a feedback form that you can send us feedback</p>
```
Comment (jaihysc) [inline Help.html]:
```suggestion
        <p>The feedback page sends you to a feedback form that you can send us feedback</p>
```

File: Help.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                <li><a href="https://forms.gle/QmQRTCUmqyDopNDF7" class="menu">Report</a></li>
+                <button type="button" id="theme_changer" class="button">Change theme</button>
+            </ul>
+        </nav>
+
+        <p>This is the help page</p>
+        <p>You can navigate to other parts of the website by pressing the menu buttons at the top on the navigation bar</p>
+        <p>The home page is just for show</p>
+        <p>The browse page is where you can browse the archived videos/pictures</p>
+        <p>The feed back page sends you to a feedback form that you can send us feedback</p>
```
Comment (jaihysc) [inline Help.html]: Using the word "feedback" three times in a sentence might be redundant.
```suggestion
        <p>The feedback page sends you to a form where you can send us feedback</p>
```

File: Js/imageLoader.js [hunk 3]
```diff
@@ -1,5 +1,15 @@
+/**
+ adds image i for each div there are in the html file
+ */
 $(document).ready( function() {
     $("div").each( function(i) {
-        $(this).append("<img src='Media/"+(++i)+".png' width='600' height='600' />");
+        $(this).append("<img src='Media/Photos/"+(++i)+".png' width='630' height='600'/>");
```
Comment (jaihysc) [inline Js/imageLoader.js]: The server does not separate between photo and videos.
Instead, look at the file extension of each file in `Media/` if it is `.png` or `.mp4`.

File: Js/videoLoader.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/**
+ *adds video i for each div there are in the html file
```
Comment (jaihysc) [inline Js/videoLoader.js]: For photos and videos, if the number of photos/videos exceed the number of divs, how will the videos which do not fit be viewed?

**Review by jaihysc — CHANGES_REQUESTED** [review]
Looking much better, left some comments regarding loading the photos and videos.

For website layout:
I think it would look better if the change theme button was aligned to the right instead of the left.

File: Js/imageLoader.js (hunk 3 above)
Comment (spin64) [reply Js/imageLoader.js]: I changed it but I don't know if it showed up here

**Review by jaihysc — APPROVED** [review]
Loading arbitrary number of images/videos is incomplete, but this can be added in a future update.

**Pull request merged**